### ISSUE TITLE
Error: TypeError: buffersPromise is not iterable

### ISSUE THREAD

**Issue by zach-aries** [alexandercerutti/passkit-generator#19]
Hello @alexandercerutti, 
First I'd like to thank you for putting this package together, it has been a great help!

I am trying to use the `pass.load()` function to bring in some dynamic resources.
But when I call the code you've used in your `download.js` examples file:
```
pass.load("https://s.gravatar.com/avatar/83cd11399b7ea79977bc302f3931ee52?size=32&default=retro", "icon.png");
pass.load("https://s.gravatar.com/avatar/83cd11399b7ea79977bc302f3931ee52?size=64&default=retro", "icon@2x.png");
```

I get the error: `Error: TypeError: buffersPromise is not iterable`

This happens with custom image resources and names as well.

I have a fix working by overwriting the pass.js file in the `node_modules` folder by adding the following:
line 71: `const buffersPromise = await this._remoteResources.reduce(async (acc, current) =>`
line 103: `let bundle = noDynList.filter((f) => !f.includes('.lproj'));`

I was going to create a pull request, but the git repo is on versions 1.6.1 where the npm package is on 1.6.2

Could you update the repo to version 1.6.2 or add a next branch?

**Comment by alexandercerutti** [comment]
Hello! Whops, my bad! Yeah, of course, I'll do it as soon as possible (more probably tomorrow Friday on evenening (GMT)).
Anyway, I wanted to notice you about my intention to remove this method, as soon as another way is available to import buffers (see #14), as resources loading is not really in the scope package. 😊
Thank you again for the report! I'll notify you once updated here.

**Comment by alexandercerutti** [comment]
Okay, I was able to push it and I saw the problem you are reporting too.
I'll wait for your PR. 😊
